rtcclient/client.py

```diff
@@ -47,7 +49,7 @@ class RTCClient(RTCBase):
     log = logging.getLogger("client.RTCClient")
 
     def __init__(self, url, username, password, proxies=None, searchpath=None,
-                 ends_with_jazz=True):
+                 ends_with_jazz=False):
```

Comment:
I don't like to change this default value. This will be a breaking change. Many users may still use this as their default value. It is incompatible.

Reply:
And it seems the value of `ends_with_jazz` is not used any more. Why?